handlers/SuiChainHandler.ts

```
@@ -644,4 +714,117 @@
       await this.bridgeSuiDeposit(deposit);
     }
   }
+
+  /**
+   * Recover deposits that are stuck in FINALIZED status by searching for their
+   * TokensTransferredWithPayload events in recent blocks
+   */
+  public async recoverStuckFinalizedDeposits(deposits: Deposit[]): Promise<void> {
+    if (!deposits || deposits.length === 0) return;
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)